gen/zap.go

```diff
@@ -26,6 +26,17 @@ import (
 	"go.uber.org/thriftrw/compile"
 )
 
+// NoZapLabel allows opt out of Zap logging for struct fields.
+// Thrift structs will use this annotation to omit it from logging, i.e.
```

Comment:
Fields of Thrift structs will use this annotation to opt-out of being logged
when that struct is logged.